### ISSUE TITLE
SignUp and Login Screen 

### ISSUE THREAD

**Issue by Astha369** [Subhradeep10/DevAcademy#9]

**Comment by Subhradeep10** [comment]
@Astha369 Go ahead, will the backend be firebase?

**Comment by Subhradeep10** [comment]
@Astha369 any updates?

**Comment by Astha369** [comment]
Yes, I'm creating the login screen, sign-up is done, just need to add firebase, and I will push all of it together.

**Comment by Subhradeep10** [comment]
@Astha369 I am closing the issue due to inactivity

**Comment by anisha-01-phoenix** [comment]
Hi @Subhradeep10, Can this issue be reopened? I would like to work on this one.
